flocker/provision/_install.py

```diff
@@ -212,6 +212,24 @@ def task_cli_pkg_test():
     return run_from_args(['flocker-deploy', '--version'])
 
 
+def wipe_yum_cache(repository):
+    """
+    Return a command which will clear all of yum's cached data for a particular
+    repository.
+
+    :param bytes repository: The name of the repository to clear.
+    """
+    return run_from_args([
+        b"yum",
+        b"--disablerepo='*'",
+        # XXX If there are shell special characters in the repository name this
+        # will break.
```

Comment:
This comment is false. `run_from_args` is designed to properly quote all the arguments.

One can observe this by seeing that the above argument is rendered as
`'--disablerepo='"'"'*'"'"''`.